p5-travel-app/webpack.dev.js

```diff
@@ -33,12 +33,13 @@ module.exports = {
     rules: [
       {
         test: "/.js$/",
-        exclude: /node_modules/,
+        // exclude: /node_modules/,
```

Comment:
remove or add comment

Reply:
done, clean it in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)